pubsub/pulsar/pulsar.go

```diff
@@ -69,6 +72,19 @@ func (p *Pulsar) Init(metadata pubsub.Metadata) error {
 	}
 	defer client.Close()
 
+	// initialize lru cache with size 100
+	c, err := lru.NewWithEvict(100, func(k interface{}, v interface{}) {
```

Comment:
Please, make this number a constant and use `10` instead. I don't think `100` is a reasonable default for number of topics being used.

Ideally, this constant should just be a default for a configuration from metadata - add a TODO for that.

Reply:
Fixed